ubyssey/templates/article/blank.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
     <main class="page">
         <div class="u-container u-container--content">
-          {{ page.get_html|safe }}
+          {{ page.html|safe }}
```

Comment:
You're right -- we definitely don't want to escape it. The difference is that we're now marking this as a "safe" string before it gets to the template:

https://github.com/ubyssey/dispatch/blob/develop/dispatch/apps/content/render.py#L16